### PR TITLE
ref(discoverTable): Use `DropdownMenu` to display cell actions

### PR DESCRIPTION
**Pull request by vuluongj20**
**Before ——**
Screenshot 2023-06-09 at 11 27 12 AM

**After ——**
Screenshot 2023-06-09 at 11 26 29 AM

### CODE REVIEW

File: static/app/views/discover/table/cellAction.tsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React, {Component} from 'react';
-import {createPortal} from 'react-dom';
-import {Manager, Popper, Reference} from 'react-popper';
```
Comment (vuluongj20) [inline static/app/views/discover/table/cellAction.tsx:3]: @evanpurkhiser yay

Comment (evanpurkhiser) [reply static/app/views/discover/table/cellAction.tsx:3]: yes yes yes yes

**Review by evanpurkhiser — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
Fantastic cleanup!!

**Review by narsaynorath — APPROVED** [review]
Awesome 🚀

**Pull request merged**